sympy/combinatorics/pc_groups.py

```diff
@@ -10,6 +10,27 @@ class PolycyclicGroup(DefaultPrinting):
     is_solvable = True
 
     def __init__(self, pc_sequence, pc_series, relative_order, collector=None):
+        """
+
+        Parameters
+        ==========
+
+        pc_sequence
+                Polycyclic sequence is formed by collecting all the
```

Comment:
I think that the explanation of how the sequence and series are formed would be better suited to the docstring of where the sequence is constructed (in perm_groups). Here we could have, for pc_series, something like "A subnormal sequence of subgroups where each factor group is cyclic", and for pc_sequence, "A sequence of elements whose classes generate the cyclic factor groups of pc_series". Relative order could be "The orders of factor groups of pc_series". Also, each parameter has a type that will come on the same line as the name after "space, colon, space" like `pc_sequence : list`.

Reply:
Suppose we've a word, what should be it's type, may be `word : FreeGroupElement`?

Reply:
I think that is the proper type.

Reply:
Ok.